data/_ui/tooltips.txt

```diff
@@ -1531,3 +1531,15 @@ tip "Show parenthesis"
 
 tip "Notify on destination"
 	`If you enter a system with a planet you must land on for a mission, plays a notification sound.`
+
+tip `Override "universal ramscoop"`
+	`Replace the "universal ramscoop" gamerule provided by the game and plugins with a custom value.`
+
+tip `"universal ramscoop" value`
+	`The value replacing the default "universal ramscoop" gamerule.`
+
+tip `Override "disabled fighters..."`
+	`Replace the "disabled fighters avoid projectiles" gamerule provided by the game and plugins with a custom value.`
```

Comment:
Somebody that would just start the game for the first time would not know what "disabled fighters avoid projectiles" means.

Reply:
Should the tooltips have the descriptions from gamerules.txt?